README.md

```diff
@@ -1966,6 +1966,24 @@ authClient.tokenManager.on('error', function (err) {
 });
 ```
 
+**NOTE**: If you manage the tokens state outside of this SDK, subscribe to `expired` event published by the `tokenManager` to get tokens properly renewed .
```

Comment:
nit: 
"If you manage the tokens state" should be "If you manage the token's state"